provisioning/roles/nginx/files/_php7_fpm.conf

```diff
@@ -0,0 +1,10 @@
+fastcgi_index index.php;
+fastcgi_pass unix:/run/php/php7.0-fpm.sock;
```

Comment:
I think it would be easier to use a variable here.

Something along the lines :

    fastcgi_pass unix:{{ fpm_socket }};
    
And add this in the defaults :

    fpm_socket: "{{ (php_version | float == 7.0) | ternary('/run/php/php7.0-fpm.sock', '/var/run/php5-fpm.sock') }}"

Then the role should also be adapted to use `template` instead of `copy` and the file moved into the `templates` directory.